### ISSUE TITLE
About the data format in deformable psroi pooling

### ISSUE THREAD

**Issue by abuzn** [Zardinality/TF_Deformable_Net#9]
in lib/networks.py, the function "deform_psroi_pool"
I think the inputs data format have been converted into [NCHW], so the output data format of this function is still [NCHW]? 
If so, the following code in "fc" function 
if data_format=="NCHW":
                    feed_in = tf.reshape(tf.transpose(input,[0,3,1,2]), [-1, dim])
will convert the data to [NWCH]？

**Comment by Zardinality** [comment]
Right, since `deform_psroi_pool` is always followed by a fc layer, I assumed the data format is transformed back to the default `NHWC` after passing through fc layer. When ported to another different network it is indeed a problem that should be awared.
